e-commerce-project/src/app/app.module.ts

```diff
@@ -13,7 +13,8 @@ import { LoginPageComponent } from './components/login-page/login-page.component
   imports: [
     BrowserModule,
     AppRoutingModule,
-    FormsModule
+    FormsModule,
+    ReactiveFormsModule
```

Comment:
Using ReactiveForms, so we need to import module first.

**_ Source description / documentation_** link: https://angular.io/guide/reactive-forms

**_Note_**: Reactive forms are one of the big topics which has a lot of reading, but you just need to see examples and and look up when you need it (mostly when you have input fields).